src/interface/meta.thrift

```diff
@@ -108,6 +111,20 @@ struct EdgeItem {
     4: common.Schema        schema,
 }
 
+struct IndexProperties {
+    1: map<string, list<string>>(cpp.template = "std::map")  fields,
```

Comment:
Is it better to append column types to each column？Therefore, the encode and decode of data are more convenient in the storage layer.